src/app/app.module.ts

```diff
@@ -152,7 +153,7 @@ export class AppModule {
 
 export const apiConfig = new Configuration({
   accessToken: '',
```

Comment:
Placeholder for now.  It apparently does not work correctly for authentication.  If it worked.  It would eliminate the need for generated files modifications.